### PR TITLE
feat: cmdline and entrypoint for worker

### PR DESCRIPTION
**Pull request by wxiwnd**
- chore: fix script of updating grpc python binding.

### CODE REVIEW

**Comment by wxiwnd** [comment]
PTAL, @sparrowhe @hedgehog098

File: grpc/supervisor.proto [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+    rpc worker_report_in (WorkerReportInRequest) returns (WorkerReportInRequest);
+    rpc update_provider_status (UpdateProviderRequest) returns (UpdateProviderStatusResponse);
+}
+
+message WorkerReportInResponse {
+    string supervisor_addr = 1;
+}
+
+message UpdateProviderStatusResponse {
+    string provider_id = 1;
```
Comment (wxiwnd) [inline grpc/supervisor.proto]: type is supposed to be int

Comment (wxiwnd) [reply grpc/supervisor.proto]: https://github.com/ProjectSoftoid/Kagami/pull/27/commits/1519436ccab0d32b37d797b13c29a1d35eab13ca

**Review by sparrowhe — APPROVED** [review]
LGTM

**Pull request merged**